README.md

```diff
@@ -1 +1,40 @@
-# at_widgets
\ No newline at end of file
+<img src="https://atsign.dev/assets/img/@dev.png?sanitize=true">
+
+### Now for some internet optimism.
+
+# at_widgets
+This is a common repository for various flutter packages that are published on [pub.dev](https://pub.dev/publishers/atsign.org/packages)
+
+These packages help to achieve the common functionalities associated with atsigns like authentication, chat, etc.
+
+# Widgets contained (which can also be found on pub.dev as linked):
+
+[at_backupkey_flutter](https://pub.dev/packages/at_backupkey_flutter)- A flutter plugin project for saving the backup key of any @sign that is being onboarded with @protocol apps. Backup key can be used to authenticate in any @protocol apps.
+
+
+[at_chat_flutter](https://pub.dev/packages/at_chat_flutter)- A flutter plugin project to provide a chat feature using atsigns and atprotocol.
+
+
+[at_common_flutter](https://pub.dev/packages/at_common_flutter)- A flutter package to provide common widgets used by other atsign flutter packages.
+
+
+[at_contacts_flutter](https://pub.dev/packages/at_contacts_flutter)- A flutter plugin project to provide ease of managing contacts for an atsign using atprotocol.
+
+
+[at_contacts_group_flutter](https://pub.dev/packages/at_contacts_group_flutter)- A flutter plugin to provide group functionality for atsign contacts.
+
+
+[at_events_flutter](https://pub.dev/packages/at_events_flutter)- A flutter plugin project to manage events.
+
+
+[at_location_flutter](https://pub.dev/packages/at_location_flutter)- A flutter plugin project to share location between two atsigns and track them on OSM (open street maps).
+
+[at_onboarding_flutter](https://pub.dev/packages/at_onboarding_flutter)- A flutter plugin project for onboarding any @sign in @protocol apps with ease. Provides QRscanner and upload key file options to authenticate.
+
+
+[atsign_authentication_helper](https://pub.dev/packages/atsign_authentication_helper)- (DISCONTINUED) A flutter plugin project to provide authentication for atsign apps. It provides both a QR scanner and feature to upload files.
+
+
```

Comment:
There is one more package called at_follows_flutter. Please check about adding this package.